### PR TITLE
Feature/initial implementation

### PR DESCRIPTION
**Pull request by dassbj01**

### CODE REVIEW

File: index.mjs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        console.log('got no token parameter');
+        // just send the input site
+        res.writeHead(302, {
+            'Location' : '/static/html/getPassword.html' 
+        })
+        res.end()
+    }else{
+        console.log('got token: ' + token);
+        unwrapSecret(token, vault_options, res)
+        //res.render('result', {message: data})
```
Comment (paulbutenko) [inline index.mjs]: leftover?

File: index.mjs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+// define the view render engine
+app.set("view engine", "pug");
+// define the views folder
+app.set("views", path.join(__dirname, "/public/html"));
+
+
+router.get('/', (req, res) => {
+    var token = req.query.token;
+    if(token == null || token == ''){
+        console.log('got no token parameter');
```
Comment (paulbutenko) [inline index.mjs:29]: leftover?

File: index.mjs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    var token = req.body.token;
+    if(token == null || token == ''){
+        console.log('got no token parameter');
+        // just send the input site
+        res.writeHead(302, {
+            'Location' : '/static/html/getPassword.html' 
+        });
+        res.end();
+    }else{
+        console.log('got token: ' + token);
```
Comment (paulbutenko) [inline index.mjs:52]: leftover?
exposing token?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  "description": "simple ui to allow users to unwrap vault tokens",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "hashicorp-vault",
+    "wrapped-secrets"
+  ],
+  "author": "bjoerndassow@googlemail.com",
```
Comment (paulbutenko) [inline package.json:13]: should be medneo mail?

File: index.mjs (hunk 2 above)
Comment (dassbj01) [reply index.mjs:29]: no. i want the logs (console.log is stdout at nodejs)

File: index.mjs (hunk 3 above)
Comment (dassbj01) [reply index.mjs:52]: no. exposing the token is no issue here .. as it is a one time token and will be exchanged against the real secret here.

File: package.json (hunk 4 above)
Comment (dassbj01) [reply package.json:13]: why?

File: index.mjs (hunk 1 above)
Comment (dassbj01) [reply index.mjs]: correct [1a1ee25](https://github.com/medneo/vault-otp-ui/pull/1/commits/1a1ee25130fbbd94850287e77c2f1ea3fce98586)

File: package.json (hunk 4 above)
Comment (paulbutenko) [reply package.json:13]: 🤷‍♀️

**Review by paulbutenko — APPROVED** [review]

**Pull request merged**